internal/cli/common.go

```diff
@@ -0,0 +1,64 @@
+package cli
+
+import (
+	"errors"
+
+	"github.com/coreos/airlock/internal/config"
```

Comment:
In Mantle & Ignition we're formatting our imports like so:

```
<stdlib>

<external lib>

<internal lib>
```

This doesn't seem to be a golang standard so not a big blocker but figured I'd bring it up.